### PR TITLE
Add react-markdown@2.5.0 w/ npm auto-update

### LINKED ISSUE #10929: [Request] Add react-markdown
**Library name:** react-markdown
**Git repository url:**  https://github.com/rexxars/react-markdown
**npm package name or url** : https://www.npmjs.com/package/react-markdown
**License (List them all if it's multiple):** MIT License
**Official homepage:** http://rexxars.github.io/react-markdown/  (Demo page)

=====================

Notes from cdnjs maintainer:
Please read the [README.md](https://github.com/cdnjs/cdnjs#cdnjs-library-repository) and [CONTRIBUTING.md](https://github.com/cdnjs/cdnjs/blob/master/.github/CONTRIBUTING.md) document first.

We encourage you to add a library via sending pull request,
it'll be faster than just opening a request issue,
since there are tons of issues, please wait with patience,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by dakshshah96**
close #10929, cc @rexxars

Pull request for issue: #10929 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/rexxars/react-markdown
 * Official website (optional, not the repository): http://rexxars.github.io/react-markdown/
 * NPM package url (optional): https://www.npmjs.com/package/react-markdown
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required): 
   - Count of watchers: 17
   - Count of stars: 838
   - Count of forks: 67
 * NPM download stats (optional):
   - Downloads in the last day: 788
   - Downloads in the last week: 17,147
   - Downloads in the last month: 75,008

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by dakshshah96** [comment]
@qwe19272375 Please add `in progress` label here.

**Comment by dakshshah96** [comment]
@qwe19272375 Please add `wait for response` label here?

**Comment by maruilian11** [comment]
@dakshshah96 Looks like the files on npm in `umd` is what we actually need. Could you please modify the PR? Thank you!

**Comment by dakshshah96** [comment]
@maruilian11 Done 👍

**Comment by dakshshah96** [comment]
@maruilian11 Sorry, missed out on that 👍 Thanks!

**Comment by dakshshah96** [comment]
@pvnr0082t Can you please review this PR?

**Comment by maruilian11** [comment]
@cdnjs/library-reviewer Please help review this PR, thanks.

**Comment by maruilian11** [comment]
@extend1994 The one looks already be minified, we don't need to do it again.

**Comment by extend1994** [comment]
@maruilian11 Sorry I though we don't need 
```
/*!
	 * pascalcase <https://github.com/jonschlinkert/pascalcase>
	 *
	 * Copyright (c) 2015, Jon Schlinkert.
	 * Licensed under the MIT License.
	 */
```

part. LGTM now. Rebasing the branch.

File: ajax/libs/react-markdown/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "markdown",
+    "react",
+    "react-component"
+  ],
+  "npmName": "react-markdown",
+  "npmFileMap": [
+    {
+      "basePath": "umd",
+      "files": [
+        "react-markdown*"
```
Comment (PeterDaveHello) [inline ajax/libs/react-markdown/package.json]: Can be `"**/*"` next time.

**Pull request merged**